lmdeploy/serve/proxy/proxy.py

```diff
@@ -447,24 +472,28 @@ async def chat_completions_v1(request: ChatCompletionRequest, raw_request: Reque
     - presence_penalty (replaced with repetition_penalty)
     - frequency_penalty (replaced with repetition_penalty)
     """
-    check_response = await node_manager.check_request_model(request.model)
-    if check_response is not None:
-        return check_response
-    node_url = node_manager.get_node_url(request.model)
-    if not node_url:
-        return node_manager.handle_unavailable_model(request.model)
-
-    logger.info(f'A request is dispatched to {node_url}')
-    request_dict = request.model_dump()
-    start = node_manager.pre_call(node_url)
-    if request.stream is True:
-        response = node_manager.stream_generate(request_dict, node_url, '/v1/chat/completions')
-        background_task = node_manager.create_background_tasks(node_url, start)
-        return StreamingResponse(response, background=background_task)
-    else:
-        response = await node_manager.generate(request_dict, node_url, '/v1/chat/completions')
-        node_manager.post_call(node_url, start)
-        return JSONResponse(json.loads(response))
+    match node_manager.serving_strategy:
```

Comment:
better not to use `match case` since it' brought in python3.10 and lmdeploy supports python>=3.8